src/screens/MyAccount.tsx

```diff
@@ -33,6 +33,8 @@ export default function MyAccount(): ReactElement {
    * On initial render, gets the details of the currently logged in user.
    */
   useEffect(() => {
+    // todo: If you are logged in and refresh the page, UserService.userId is no longer remembered. We need
+    //  to have AuthUserContext somehow remember the id of the user even after refreshing
```

Comment:
I added this new todo. Currently, if I sign in and select the option to stay sign in, this is equivalent to configuring firebase auth as `this.auth.setPersistence(firebase.auth.Auth.Persistence.LOCAL);` because we are persisting the auth locally, so even if I close the tab it will be remembered if I open in another tab.

This is great, but not useful if we cannot also remember the user information such as the `id` in local storage too.

Reply:
Good point. I think we can make a request to the backend in AuthUserContext when it first runs to get the user id if the user is already signed in.